### PR TITLE
Issue 581: Add service units field to project view

### PR DESCRIPTION
**Pull request by Hzaakk**
image

### CODE REVIEW

**Comment by matthew-li** [comment]
@Hzaakk Thanks, but this should be on the home page! Also, given that the column name is "Service Units", we don't need the label in each row.

**Comment by matthew-li** [comment]
From today's meeting, we'll keep this on the project list views too.

File: coldfront/core/project/templates/project/project_list.html [hunk 1]
```diff
@@ -118,10 +118,21 @@ <h1>{{ PROGRAM_NAME_SHORT }} Cluster Projects</h1><hr>
             <i class="fas fa-sort-down"></i>
           </a>
         </th>
+        <th scope="col" class="text-nowrap">
```
Comment (matthew-li) [inline coldfront/core/project/templates/project/project_list.html:121]: I get an error when attempting this--do you? I'm fine with not having it sortable.

File: coldfront/core/project/views.py [hunk 2]
```diff
@@ -443,6 +443,17 @@ def get_context_data(self, **kwargs):
         context['user_agreement_signed'] = \
             access_agreement_signed(self.request.user)
 
+        compute_allocation_information = []
+        for project in project_list:
+            try:
+                information = get_project_compute_allocation(project).get_information
+                information = information[len('Service Units: '):-len(' <br>')]
+                compute_allocation_information.append(information if information else 'N/A')
```
Comment (matthew-li) [inline coldfront/core/project/views.py:451]: Any reason to do this differently from the other view? `project.compute_allocation_information = ...`

**Review by matthew-li — CHANGES_REQUESTED** [review]
Overall, looks fine, thanks. Had an issue with sorting, and a question.

File: coldfront/core/project/templates/project/project_list.html (hunk 1 above)
Comment (Hzaakk) [reply coldfront/core/project/templates/project/project_list.html:121]: Yeah I removed it

**Comment by matthew-li** [comment]
@Hzaakk I made a dedicated function to reduce duplication. It fetches the values directly, instead of calling `Allocation.get_information`.

The main difference you'll notice is that "Inactive" projects will render N/A instead of zero.

Can you review the code / website functionality, and lmk if you run into any issues?

**Comment by Hzaakk** [comment]
@matthew-li looks good and works on my end

**Pull request merged**